client/src/containers/header/index.tsx

```diff
@@ -11,52 +12,55 @@ import menuSignout from 'assets/icons/header/menu/logout.svg';
 import { ROUTES } from 'constants/routes';
 import { useHistory } from 'react-router-dom';
 
-
-
 const Header = () => {
-    const history = useHistory();
-    const dispatch = useDispatch();
-    const handleLogout = async () => {
-        dispatch(actions.logoutUser());
-    };
-    const { user } = useAppSelector((state) => state.auth.userData);
+	const history = useHistory();
```

Comment:
there is a historyHelper that you could use:
`import historyHelper from 'helpers/history.helper';`